benchmark/stats/histogram.go

```diff
@@ -4,14 +4,18 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"log"
 	"math"
 	"strconv"
 	"strings"
-	"time"
 )
 
-// HistogramValue is the value of Histogram objects.
-type HistogramValue struct {
+// Histogram accumulates values in the form of a histogram with
+// exponentially increased bucket sizes.
+// The first bucket (with index 0) is [0, n) where n = baseBucketSize.
```

Comment:
[HistogramOptions.MinValue, HistogramOptions.MinValue + n) and move this to the comments of HistogramOptions.